src/docs/BUILD.md

```diff
@@ -3,85 +3,200 @@
 > **NOTE**: There is a known Python API issue with `ov::Tensor`. The issue is reproduced when building OpenVINO GenAI from sources while using OpenVINO from archives. Using `ov::Tensor` with OpenVINO GenAI fails. Possible errors: `TypeError: generate(): incompatible function arguments.`, `TypeError: __init__(): incompatible constructor arguments.`, `TypeError: Unregistered type : ov::Tensor`.
 The preferred approach is to build both OpenVINO and OpenVINO GenAI from sources using the same build environment. Or to install prebuilt OpenVINO GenAI from [distribution channels](https://docs.openvino.ai/2024/get-started/install-openvino.html).
 
-## Build for Linux Systems
+## Software Requirements
 
-### Software Requirements 
+### Linux
 
 - [CMake](https://cmake.org/download/) 3.23 or higher
 - GCC 7.5 or higher
 - Python 3.8 or higher
+- Git
```

Comment:
Please include changes from https://github.com/openvinotoolkit/openvino.genai/pull/652. Most of them aren't applicable anymore, but there should be `\[NSIS](https://sourceforge.net/projects/nsis/)` added at least. Feel free to do that in a separate PR.